docs/running_lbann.rst

```diff
@@ -1,96 +1,434 @@
 .. role:: bash(code)
           :language: bash
+.. role:: python(code)
+          :language: python
```

Comment:
I think you also need to add a `role` for `proto` because it is used on line 395.

Reply:
Oops, that's been fixed.